apps/src/blockly/addons/cdoXml.js

```diff
@@ -7,7 +7,7 @@ export default function initializeBlocklyXml(blocklyWrapper) {
   blocklyWrapper.Xml.originalBlockToDom = blocklyWrapper.Xml.blockToDom;
   blocklyWrapper.Xml.blockToDom = function(block) {
     const blockXml = blocklyWrapper.Xml.originalBlockToDom(block);
-    if (!block.canDisconnectFromParent_) {
+    if (block.canDisconnectFromParent_ === false) {
```

Comment:
block.canDisconnectFromParent_ shouldn't return anything other than a boolean value so using a falsy operator in this context should be ok

Reply:
Agreed. @mikeharv was there a reason this needed to change?